src/presentation/ui/page/databases.templ

```diff
@@ -0,0 +1,299 @@
+	}
+
+	return tabHeaderItems
+}
+
+templ DatabasesTabs(selectedDbTypeSummary presenterDto.SelectedDatabaseTypeSummary) {
+	@componentStructural.TabHeader(
+		getDatabasesTabHeaderItems(selectedDbTypeSummary.Type), "changeSelectedDatabaseType",
+	)
+	<!-- DatabaseTabContent -->
+	<div class="bg-os-800 p-4 rounded-b-xl rounded-r-xl">
+		<div class="px-4 bg-os-500 rounded-lg space-y-3">
+			if !selectedDbTypeSummary.IsInstalled {
+				<div class="p-4 bg-os-500 rounded-r-lg">
+					<i class="ph-duotone ph-warning align-middle text-xl text-yellow mr-3"></i>
+					<span class="mr-3">The<p class="inline-flex mx-1 text-speedia-300 font-bold">{ selectedDbTypeSummary.Type.String() }</p>service is not installed yet.</span>
+					@componentForm.SubmitButton("Schedule "+selectedDbTypeSummary.Type.String()+" service installation", "ph-caret-line-down", "installDatabaseService('"+selectedDbTypeSummary.Type.String()+"')", true)
+				</div>
+			} else {
+				@componentDatabases.InstalledDatabaseTypeSummaryTable(selectedDbTypeSummary.Databases)
+				<div class="bg-os-500 p-0.5"></div>
+			}
+		</div>
+	</div>
+}
+
+templ CreateDatabaseForm(selectedDatabaseType valueObject.DatabaseType) {
+	<form
+		hx-post={ "/api/v1/database/" + selectedDatabaseType.String() }
+		hx-indicator="#loading-overlay"
+		hx-swap="none"
+	>
+		<div class="my-6">
+			@componentForm.InputField("text", "dbName", "Database Name", "database.name", false)
+		</div>
+		@componentForm.DeactivatableSubmitButton(
+			"", "Create", "ph-check-fat", "closeCreateDatabaseModal()", "shouldDisableCreateDatabaseSubmitButton", false,
+		)
+	</form>
+}
+
+templ CreateDatabaseModal(selectedDatabaseType valueObject.DatabaseType) {
+	@componentStructural.Modal(
+		"Create Database", "isCreateDatabaseModalOpen", "closeCreateDatabaseModal()", "",
+	) {
+		@CreateDatabaseForm(selectedDatabaseType)
+	}
+}
+
+func getOnlyDatabasesNames(selectedDatabaseTypeDatabases []entity.Database) []string {
+	databasesNames := []string{}
+	for _, database := range selectedDatabaseTypeDatabases {
+		databasesNames = append(databasesNames, database.Name.String())
+	}
+
+	return databasesNames
+}
+
+templ CreateDatabaseUserForm(
+	selectedDatabaseType valueObject.DatabaseType,
+	selectedDatabaseTypeDatabases []entity.Database,
+) {
+	<div class="my-6 space-y-6">
+		@componentForm.SelectInput(
+			"dbName", "Database Name", "database.name", getOnlyDatabasesNames(selectedDatabaseTypeDatabases), false,
+		)
+		@componentForm.InputField("text", "username", "Username", "databaseUser.username", false)
+		<div class="space-y-2">
+			@componentForm.PasswordInput(
+				"password", "Password", "databaseUser.password", false,
+			)
+			<div class="flex justify-end">
+				@componentStructural.LabeledButton(
+					"generate random password", "ph-arrows-clockwise", "os-800", "os-200",
+					"setDatabaseUserPassword('"+infraHelper.GenPass(16)+"')", true,
```

Comment:
What? Hahaha. Dude, you must do this via JS. The user must have more than one choice.

Reply:
Done!